iina/MainWindowController.swift

```diff
@@ -20,6 +20,11 @@ fileprivate let OSCTopMainViewMarginTopInFullScreen: CGFloat = 6
 fileprivate let PlaylistMinWidth: CGFloat = 240
 fileprivate let PlaylistMaxWidth: CGFloat = 400
 
+fileprivate let UIAnimationDuration: TimeInterval = 0.25
```

Comment:
`TimeInterval` is a typealias for `Double`, so you don't need to explicitly specify the type.